umake/decompressor.py

```diff
@@ -28,7 +28,6 @@
 import tarfile
 import zipfile
 
-
```

Comment:
If you want to do this kind of cleanage, please do it in one commit, and in another branch. That doesn't have anything to do with the current matter.